variants/ET_Board/pins_arduino.h

```diff
@@ -0,0 +1,47 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        40
+#define NUM_ANALOG_INPUTS       7
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(esp32_adc2gpio[(p)]):-1)
+#define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 34)
+
+static const uint8_t TX = 34;
+static const uint8_t RX = 35;
+
+static const uint8_t SDA = 33;
+static const uint8_t SCL = 36;
+
+static const uint8_t SS    = 29;
+static const uint8_t MOSI  = 37;
+static const uint8_t MISO  = 31;
+static const uint8_t SCK   = 30;
+
+static const uint8_t A0 36   // BUILTIN_Potentiometer
```

Comment:
you have forgotten the equal sign and period at the end. This will not compile :)

`static const uint8_t A0 = 36;`

Reply:
thank you for your advice.